solutions/io.lionweb.mps.cmdline/readme.md

```diff
@@ -182,7 +182,7 @@ lionwebVersion=0.2.10
 lionwebRelease=2023.1
 mpsVersionSuffix=2021.1
 mpsVersion=2021.1.4
-comSpecificlanguagesMpsVersion=1.6.0
```

Comment:
If all three projects work with the newer version, I'd go with that.

Reply:
OK, I'll try that out – 1st in a separate branch to check whether local+CI works

Reply:
That works: https://github.com/LionWeb-io/lionweb-mps/actions/runs/13390082812